libPhoneNumber-iOS.podspec

```diff
@@ -11,12 +11,13 @@ DESC
   s.authors      = { "iziz" => "zen.isis@gmail.com", "hyukhur" => "hyukhur@gmail.com" }
   s.source       = { :git => "https://github.com/iziz/libPhoneNumber-iOS.git", :tag => s.version.to_s }
   s.libraries 	 = 'z'
+  s.module_name  = "libPhoneNumberiOS"
```

Comment:
Can you separate the module change and the Cocoapods fix into 2 separate PRs? As one is a fix, and another one is a breaking change. Thanks!